workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/pages/GetAppPage/components/SelectPaymentMethod/components/TrialTimeLine/TrialTimeline.tsx

```diff
@@ -0,0 +1,83 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+/* eslint-disable react/no-unescaped-entities */
+
+import circleFill from '../../../../../../assets/icons/circle_fill_icon.svg';
+import radioSelected from '../../../../../../assets/icons/radio_button_checked_2_icon.svg';
+import timeline from '../../../../../../assets/images/timeline.png';
+
+import './TrialTimeline.scss';
+
+export function TrialTimeline() {
+	const months = [
+		'January',
+		'February',
+		'March',
+		'April',
+		'May',
+		'June',
+		'July',
+		'August',
+		'September',
+		'October',
+		'November',
+		'December',
+	];
+
+	const date = new Date();
+
+	date.setTime(date.getTime() + 30 * 24 * 60 * 60 * 1000);
```

Comment:
We need to know what it is and if possible to move it to a util file.

Reply:
Hey @juliocezar96 can you do this modification in the trial component, please? This is related with you activity